5.sc_Morphology_Logistic_Regression/1.train_models/README.md

```diff
@@ -0,0 +1,23 @@
+# Training models
+
+Each logistic regression model run with parameters from the config file for each type of model:
+Model 1: Logistic regression - Binary Classifier
+Model 2: Logistic regression - Multiclass Classifier
```

Comment:
```suggestion
- Model 1: Logistic regression - Binary Classifier
- Model 2: Logistic regression - Multiclass Classifier
```

I assume this is an okay format? I think it will make the readme a bit more readable.